python/benchmark/run-for-all-releases.py

```diff
@@ -0,0 +1,19 @@
+import json
+import os
+from urllib.request import urlopen
+
+import tqdm
+from distutils.version import StrictVersion
+
+
+def versions(package_name):
+    url = f"https://pypi.org/pypi/{package_name}/json"
+    data = json.load(urlopen(url))
+    return sorted(data["releases"].keys(), key=StrictVersion)
+
+
+if __name__ == "__main__":
+    versions = [v for v in versions("tskit") if "a" not in v and "b" not in v]
+    for v in tqdm.tqdm(versions):
+        os.system(f"pip install tskit=={v}")
```

Comment:
I think it's probably worth automating in a venv here, so that the devs environment doesn't get messed up by running this.